packages/x-components/src/components/modals/close-x-modal.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <BaseEventsModalClose v-on="$listeners" class="x-modal" :closing-event="closingEvent">
```

Comment:
I think we are using `camelCase` for the `props`. I think webstorm's autocomplete uses the `kebab-case` approach. Can you refactor them?

```suggestion
  <BaseEventsModalClose v-on="$listeners" class="x-modal" :closingEvent="closingEvent">
```